source/vir/src/sst_to_air.rs

```diff
@@ -452,6 +445,60 @@ fn one_stmt(stmts: Vec<Stmt>) -> Stmt {
     if stmts.len() == 1 { stmts[0].clone() } else { Arc::new(StmtX::Block(Arc::new(stmts))) }
 }
 
+// convert the sst expression into bv air expression
+fn exp_to_bv_expr(state: &State, exp: &Exp, parent_width: u32) -> (Expr, u32) {
+    let mut parent_width = parent_width;
+
+    match &exp.x {
+        ExpX::Const(crate::ast::Constant::Nat(s)) => {
+            assert!(parent_width != 0);
+            // Nat constant will get an inferred bitwidth from the parent
+            // the width is needed when printing bv constants
```

Comment:
Just to make this clearer, can we add a period at the end of the last line and capitalize "The width..."?

Reply:
Sure.

Reply:
I think you don't need `parent_width` at all, since `exp.typ` will give you the width.

Reply:
Ah good to know, that would actually simplify things a lot here.